guides.md

```diff
@@ -190,6 +191,43 @@ We'll now send your first email to Mailinator, a cloud inbox that's helpful for
 9. Visit [the corresponding cloud inbox](https://www.mailinator.com/v3/index.jsp?zone=public&query=hyphacoop) and confirm your message has arrived, It might take a few minutes.
 10. Don’t forget to email Infra WG as mentioned in onboarding email.
 
+## Expense Reimbursement
+
+Employees should try to submit eligible expenses in the same quarter they are incurred.
+The Finance WG reviews submitted expenses before each pay period, and pays out reimbursement amounts on the next pay day.
+
+### Submitting an expense
+
+1. Upload the expense receipt to [Employee Expenses](https://link.hypha.coop/receipts) with file name `xx-YYYY-MM-DD.ext` where `xx` is the initials of the member and `ext` the file extension, then make note of the URL.
```

Comment:
See comment above about naming and link matching

Reply:
Also do you need CC statement? Make clear it is an itemized receipt?

Reply:
I don't think we need CC statement. Do you think we do?

Reply:
I think you should get a copy for foreign currency

Reply:
@dcwalk is this a legal requirement or an internal process you recommend?
@YurkoWasHere your thoughts?